crates/miden-batch-prover/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "miden-batch-prover"
```

Comment:
I wonder if we should name this `miden-tx-batch-prover` to indicate that we are proving batches of transactions.